qa/tasks/salt_manager.py

```diff
@@ -27,6 +28,19 @@
 master_role = 'client.salt_master'
 
 
+def systemctl_status(self, service_name):
+    """
+    Caveat: only works for units ending in ".service"
+    """
+    def service_status_cmd(remote, service, lines=0):
```

Comment:
I believe you missed something here:
1) service_status_cmd should have signature (service, lines=0) 
2) systemctl_status should have signature (self, remote, service_name)

Reply:
oops - fixed